rust-lang/crates.io
# Move scrolling to component

## Tasks


## QA

- Try several different transitions and make sure that the page always scrolls to the top.
